lib/yoti/dynamic_share_service/policy/wanted_attribute.rb

```diff
@@ -23,6 +28,7 @@ def as_json(*_args)
         }
         obj[:derivation] = @derivation if derivation
         obj[:accept_self_asserted] = @accept_self_asserted if accept_self_asserted
+        obj[:constraints] = @constraints.as_json unless constraints.empty?
```

Comment:
I needed to change this for the Sinatra example:
```ruby
obj[:constraints] = @constraints.map {|constraint| constraint.as_json } unless constraints.empty?
```

Perhaps we need some extra test coverage for adding constraints in `wanted_attribute_spec.rb`?

Reply:
expect(attribute.json).to eql '{"... should be covering it? Tried it in Sinatra and it worked? I think I'm missing something.

Reply:
Oh wait. I just realised my branch is ahead by 1 commit. 'THat'll do it'.